packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -459,6 +459,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     if (ev.which === openKey) {
       this._onItemSubMenuExpand(item, ev.currentTarget as HTMLElement);
+      ev.preventDefault();
```

Comment:
I agree it's not a super common pattern, but I think it's cleaner than using stopPropagation. Of course, all modern browsers support the event capture phase, so things like telemetry can still be achieved regardless of stopPropagation being called. The alternative to the change in this PR is to add stopPropagation (in addition to preventDefault, perhaps?)